### ISSUE TITLE
PMID:19487461

### ISSUE THREAD

**Issue by Antonialock** [pombase/fypo#3310]
cell cycling without mitosis 
these cells undergo G1 transcription, a seemingly normal Sphase (no region specific amplifications, relatively normal replication origin firing usage etc) and can only reinitiate replication once cell size per genome number is minimal size....they just don't do mitosis

**Comment by ValWood** [comment]
isn't this "endoreplication" ? 

is it a cdc13 mutant?

**Comment by Antonialock** [comment]
yes it is cdc13

there is no phenotype matching "endoreplication"

**Comment by ValWood** [comment]
no I was just trying to figure out what  you are trying to describe ;). We must have  *something*
for this??? Midori will let you know on monday....

**Comment by ValWood** [comment]
increased diploidization or something?

**Comment by Antonialock** [comment]
they are not really diploids...a diploid has 2 parents? and these guys just seem to replicate their DNA...but in a "controlled fashion" (orderly series of events of enough growth to trigger S phase, then turn on the transcriptional program for S phase, then do S phase, repeat to 32x genome). - I looked at another paper where the mutants (not cdc13) kept replicating their genome but in a more uncontrolled manner (e.g. origins fired more than once etc)

**Comment by ValWood** [comment]
that's how we have used it...

"increased spontaneous diploidization"

A cell phenotype observed in the vegetative growth phase of the life cycle in which heterothallic haploid cells spontaneously form cells with diploid DNA content at a higher frequency than normal.


I don't think a diploid needs to have 2 parents? It's just describing DNA ploidy?

**Comment by Antonialock** [comment]
maybe....in any case this mutant doesnt contain diploid dna content...it goes up to 32x..

**Comment by ValWood** [comment]
increased spontaneous polyploidization

**Comment by ValWood** [comment]
@JackyVH  may have some input. that sounds sensible to me though...

**Comment by JackyVH** [comment]
I think  Antonia description of 'cell cycling without mitosis'  best describes what is happening the cells go through repeated, what look like normal periodic S phases,which are dependent on G1 transcription and reaching the minimal cell size but they don't do mitosis

**Comment by mah11** [comment]
Is there any difference between "cell cycling without mitosis" and FYPO:0001425? If so, what are the distinguishing features?

**Comment by JackyVH** [comment]
I  don't think think FYPO:0001425 is wrong but implies that the regulation of DNA replication is abnormal where as in 'cell cycling with out mitoses' implies that the regulation is normal but misses out mitosis. I can see that this could  therefore be abnormal regulation of DNA replication but to me it means the cell can't do mitosis but everything else is pretty normal. The loss of cdc13 means the cell cannot enter mitosis and then G1  but is in  a "G1 state" and so does what it normally does in G1  and goes on the replicate its DNA when it reaches the right size for its ploidy. Does this make sense? I have to say I hadn't  really thought of it this way before.

**Comment by ValWood** [comment]
Ah, this  is why I always struggled with the "negative regulation of DNA replication"

https://github.com/pombase/fypo/issues/304
https://github.com/pombase/fypo/issues/1702

**Comment by mah11** [comment]
> I don't think think FYPO:0001425 is wrong but implies that the regulation of DNA replication is abnormal

That's correct. In eukaryotes, normal regulation of DNA replication initiation includes restricting it to once per cell cycle. 

> where as in 'cell cycling with out mitoses' implies that the regulation is normal but misses out mitosis.

If initiation re-occurs without an intervening mitosis, then regulation of initiation is not normal. (There are exceptions, in systems where endoreduplication is normal, but they're usually specialized cells or tissues in multicellular organisms, and not relevant to pombe.)

**Comment by mah11** [comment]
I've actually got a better idea about how to distinguish two types of FYPO:0001425 from PMID:20736315 (the paper in #3309), so I added these:

id: FYPO:0006575
name: abnormal negative regulation of mitotic DNA replication initiation resulting in complete rereplication, with otherwise normal S phase
def: A regulation phenotype observed in the vegetative growth phase of the life cycle in which abnormal negative regulation of the initiation of mitotic DNA replication results in one or more rounds of rereplication of the entire genome, but takes place during S phases that appears nearly normal apart from occurring without intervening mitoses. Origin usage is normal, rounds of DNA synthesis are correlated with G1/S phase gene expression and cell mass increase, and the genome is replicated evenly.
synonym: "cell cycling without mitosis" RELATED [PomBase:al,PomBase:jh]
is_a FYPO:0001425 ! abnormal negative regulation of mitotic DNA replication initiation resulting in complete rereplication

id: FYPO:0006576
name: abnormal negative regulation of mitotic DNA replication initiation resulting in complete rereplication, with abnormal S phase
def: A regulation phenotype observed in the vegetative growth phase of the life cycle in which abnormal negative regulation of the initiation of mitotic DNA replication results in one or more rounds of rereplication of the entire genome, and other features of S phase are abnormal. Rounds of replication are not correlated with G1/S phase gene expression, and origin usage is altered.
synonym: "abnormal negative regulation of mitotic DNA replication initiation resulting in continuous DNA synthesis" RELATED [PomBase:al]
is_a FYPO:0001425 ! abnormal negative regulation of mitotic DNA replication initiation resulting in complete rereplication


They're is_a FYPO:0001425 because regulation of replication initiation is abnormal in both cases, as noted in my previous comment.

I've used FYPO:0001425 to annotate some cdc18 phenotypes; I would not move those annotations because they're from slightly older papers that only show DNA content by FACS. They did not look at cell size or G1/S gene expression, and so didn't gather the data that would be necessary to choose one or the other sub-type. (Same goes for annotations to FYPO:0001426.)

edit file: 40fba236182fa955ab21d3c0212ef93056fb31b3
release: 6a623d18fa658d9addc4479b145f8c08c6938aba
